README.md

```diff
@@ -2249,7 +2249,7 @@ Other Style Guides
     }
     ```
 
-  <a name="control-statement--value-selection"></a>
+  <a name="control-statements--value-selection"></a>
```

Comment:
While this appears to be a typo from the [original commit](c8b116410), to prevent links from breaking, we need to keep the singular one forever.

Either we need to fix the links to point to the plural one, or, we should add an anchor tag here for both.